tests/features/joinup_event/add_event.solution.feature

```diff
@@ -0,0 +1,64 @@
+@api
+Feature: "Add event" visibility options.
+  In order to manage events
+  As a solution member
+  I need to be able to add "Event" content through UI.
+
+  Scenario: "Add event" button should not be shown to normal members, authenticated users and anonymous users.
+    Given the following solutions:
+      | title           | logo     | banner     |
+      | Ragged Tower    | logo.png | banner.jpg |
+      | Prince of Magic | logo.png | banner.jpg |
+    And the following collection:
+      | title      | Collective Ragged tower       |
+      | logo       | logo.png                      |
+      | banner     | banner.jpg                    |
+      | affiliates | Ragged Tower, Prince of Magic |
+
+    When I am logged in as an "authenticated user"
+    And I go to the homepage of the "Ragged Tower" solution
+    Then I should not see the link "Add event"
+
+    When I am an anonymous user
+    And I go to the homepage of the "Ragged Tower" solution
+    Then I should not see the link "Add event"
+
+    When I am logged in as a "facilitator" of the "Ragged Tower" solution
+    And I go to the homepage of the "Ragged Tower" solution
+    Then I should see the link "Add event"
```

Comment:
Are you running maybe the test with selenium? because it's not failing in my environment and it will fail with selenium as the link is css hidden until the user presses the + button.